lib/components/Modal/index.tsx

```diff
@@ -11,19 +11,21 @@ interface OwnProps {
   title?: string,
   description?: string,
   onCancel?: () => void,
+  onWrapperCancel?: () => void,
   cancelContent?: string,
   onContinue?: () => void,
   continueContent?: string,
   continueDanger?: boolean,
   applicationIcon?: string,
   themeColor?: string,
   isLoading?: boolean,
+  disableWrapperClick?: boolean,
 }
 
 const styles = (theme: ThemeTypes) => createStyles({
   container: {
     position: 'relative',
-    width: 450,
+    width: 400,
```

Comment:
I think that it once was `400` but we increased it because of one of our Modal (don't remember which one or why  though ...)

Reply:
We checked with the design team, it was `350` before, and this new value doesn't seem to cause any troubles (and matches better). I believe we can give it a try like that 👍